Fix error messages in extension log when quick evalling library files with CodeQL CLI v2.10.0 (#1412)
github/vscode-codeql: extensions/ql-vscode/CHANGELOG.md

## [UNRELEASED]

- Fix a bug where quick queries cannot be compiled if the core libraries are not in the workspace. [#1411](https://github.com/github/vscode-codeql/pull/1411)

> - Fix a bug where quick evaluation of library files would print an error message to the log when using CodeQL CLI v2.10.0. [#1412](https://github.com/github/vscode-codeql/pull/1412)

Reviewer: ```suggestion
- Fix a bug where quick evaluation of library files would display an error message when using CodeQL CLI v2.10.0. [#1412](https://github.com/github/vscode-codeql/pull/1412)
```
This is a bit more accurate, since it wasn't just in the logs but also in a pop-up.
